cardano-dev/kevm/testnet-web3/kevm-web3-testnet-example.js

```diff
@@ -1,7 +1,8 @@
 /*
-FIRST: copy and run the following npm install lines into your console to install all all required modules (if not allready installed)
+FIRST: copy and run the following npm install lines into your console to install all required modules (if not allready installed)
 
 npm install web3
+npm install crypto
```

Comment:
Ok, I think that the `require('crypto')` part is needed, but the install - not. The [page I was linking last time](https://www.npmjs.com/package/crypto) says that this is now a "built-in" package which mean that every `node` installation already contains it, by default. But you still need to require it from a script.

For example, I have never **installed** it explicitly, but this code works for me:

![](https://i.imgur.com/OhOZWXO.png)